scripts/seed-database/container.sh

```diff
@@ -33,20 +33,20 @@ for table in "${tables[@]}"; do
 done
 
 if [[ ${RESET} == "reset_flows" ]]; then
-  psql ${LOCAL_PG} --command="TRUNCATE TABLE flows CASCADE;"
+  cat write/truncate_flows.sql > sync.sql
 fi
 
 if [[ ${INCLUDE_PUBLISHED_FLOWS} == "include_published_flows" ]]; then
   psql --quiet ${REMOTE_PG} --command="\\copy (SELECT DISTINCT ON (flow_id) id, data, flow_id, summary, publisher_id, created_at FROM published_flows ORDER BY flow_id, created_at DESC) TO '/tmp/published_flows.csv' (FORMAT csv, DELIMITER ';');"
   echo published_flows downloaded
```

Comment:
while we're adjusting the sync, i personally don't agree that there's ever a good reason to _not_ include published flows and would be in favor of removing this conditional - which adds extra complexity like updating synced columns in two places for this particular table. 

understand you didn't introduce it here, but would help simplify concatenation i think / see other comment!